src/pages/LandingPage.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import HeroSection from '../components/client/landing/heroSection/'
```

Comment:
remove "/" at the end.
example:
import HeroSection from '../components/client/landing/heroSection'
Do the same thing for others